Jarolift_MQTT.ino

```diff
@@ -330,6 +330,7 @@ void loop()
 
   // Check if RX buffer is full
   if ((lowbuf[0] > 3650) && (lowbuf[0] < 4300) && (pbwrite >= 65) && (pbwrite <= 75)) {     // Decode received data...
+    WriteLog("[INFO] - received data", true);
```

Comment:
Logging all received radio data is something I will not commit. (Line 333)
In my house something causes to receive some noise every few minutes, this will float the log with useless messages. When we have one day a check for valid received data, then we can log this too.

Reply:
I had similar issues, and that's why I wanted a timestamped logging. This output appears only on the serial port and does not load the log file. Because logging new_line false generates a) no newline and causes b) that the message does not come into the log file.

Reply:
`WriteLog` logs to serial console and the web log.
If you insist on logging all received data to web log, make it configurable through the web interface. And such a change is worth a separate PR.